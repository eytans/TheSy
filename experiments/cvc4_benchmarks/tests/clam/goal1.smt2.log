19:41:44 [WARN] Collecting statistics
19:41:44 [WARN]  ex is: zero
19:41:44 [WARN]  ex is: (succ zero)
19:41:44 [WARN]  ex is: (succ (succ zero))
19:41:44 [WARN] Running exploration without proof mode, but goals were given
19:41:44 [WARN] Proof mode: true
19:41:44 [WARN]  ex is: zero
19:41:44 [WARN]  ex is: (succ zero)
19:41:44 [WARN]  ex is: (succ (succ zero))
19:41:44 [WARN] Running exploration without proof mode, but goals were given
19:41:44 [WARN] Proof mode: true
19:41:44 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:41:44 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:41:46 [WARN] generalized case_split proof as: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) -- (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:41:46 [WARN] proved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:41:46 [WARN] proved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:41:46 [WARN] proved: (double (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus (double ?ts_ph_Nat_0) (double ?ts_ph_Nat_1))
19:41:46 [WARN] proved: (plus (double ?ts_ph_Nat_0) (double ?ts_ph_Nat_1)) => (double (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:41:46 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:41:46 [WARN] proved: (succ (succ ?ts_ph_Nat_0)) => (succ (plus ?ts_ph_Nat_0 (succ zero)))
19:41:46 [WARN] proved: (succ (plus ?ts_ph_Nat_0 (succ zero))) => (succ (succ ?ts_ph_Nat_0))
19:41:46 [WARN] Found all lemmas
