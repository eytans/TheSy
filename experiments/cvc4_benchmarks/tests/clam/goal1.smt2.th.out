19:41:44 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(double zero) => zero
zero => (double zero)
(double (succ ?n)) => (succ (succ (double ?n)))
(succ (succ (double ?n))) => (double (succ ?n))
19:41:44 [0m[33m[WARN] [0m ex is: zero
19:41:44 [0m[33m[WARN] [0m ex is: (succ zero)
19:41:44 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:41:44 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:41:44 [0m[33m[WARN] [0mProof mode: true
19:41:44 [0m[33m[WARN] [0m ex is: zero
19:41:44 [0m[33m[WARN] [0m ex is: (succ zero)
19:41:44 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:41:44 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:41:44 [0m[33m[WARN] [0mProof mode: true
19:41:44 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:41:44 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:41:46 [0m[33m[WARN] [0mgeneralized case_split proof as: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) -- (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:41:46 [0m[33m[WARN] [0mproved: (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2) => (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2))
19:41:46 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (plus ?ts_ph_Nat_1 ?ts_ph_Nat_2)) => (plus (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1) ?ts_ph_Nat_2)
19:41:46 [0m[33m[WARN] [0mproved: (double (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus (double ?ts_ph_Nat_0) (double ?ts_ph_Nat_1))
19:41:46 [0m[33m[WARN] [0mproved: (plus (double ?ts_ph_Nat_0) (double ?ts_ph_Nat_1)) => (double (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:41:46 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
19:41:46 [0m[33m[WARN] [0mproved: (succ (succ ?ts_ph_Nat_0)) => (succ (plus ?ts_ph_Nat_0 (succ zero)))
19:41:46 [0m[33m[WARN] [0mproved: (succ (plus ?ts_ph_Nat_0 (succ zero))) => (succ (succ ?ts_ph_Nat_0))
19:41:46 [0m[33m[WARN] [0mFound all lemmas
done in 2043
